health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/viewVillage.js

```
@@ -232,16 +237,16 @@ const VillageView = () => {
 
             </div>
             {/* commenting becuase some css is not working inside the component*/}
-            {/* <ActionBar
+            <ActionBar
                 actionFields={[
-                    <Button icon="ArrowBack" label="Back" onClick={function noRefCheck() { }} type="button" variation="secondary" />,
+                    <Button icon="ArrowBack" label={t(`HCM_MICROPLAN_VIEW_VILLAGE_BACK`)} onClick={function noRefCheck() { }} type="button" variation="secondary" />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)